bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/api/model/PlugwiseHAControllerRequest.java

```diff
@@ -224,18 +219,24 @@ private String getContent() throws PlugwiseHAException {
         return content;
     }
 
-    private ContentResponse getContentResponse() throws PlugwiseHAException {
+    private ContentResponse getContentResponse(int retries) throws PlugwiseHAException {
         Request request = newRequest();
         ContentResponse response;
 
-        if (logger.isTraceEnabled()) {
-            logger.trace(">> {} {}", request.getMethod(), request.getURI());
-        }
+        this.logger.debug("Performing API request: {} {}", request.getMethod(), request.getURI());
 
         try {
             response = request.send();
-        } catch (TimeoutException | InterruptedException e) {
+        } catch (InterruptedException e) {
+            this.logger.trace("InterruptedException occured {} {}", e.getMessage(), e.getStackTrace());
```

Comment:
Please add this line just after logging:
Thread.currentThread().interrupt();

Reply:
Added it